hubspot/discovery/discovery_base.py

```diff
@@ -1,4 +1,9 @@
-import pkg_resources
+import sys
+
+if sys.version_info >= (3, 8):
+    from importlib import metadata as importlib_metadata
+else:
+    import importlib_metadata
```

Comment:
Another option here is to use the “Easier to Ask Forgiveness Than Permission” (EAFP) style, which is arguably more pythonic.

```suggestion
try:
    from importlib import metadata as importlib_metadata
except ImportError:
    import importlib_metadata
```

Reply:
IMO better to keep it like this so it can be found more easily when Python 3.7 support is removed.

Arguably there shouldn't be Python 3.7 support anyway as it's been EOL for nearly a year now. Not sure what the maintenance policy for this package is but if it's for non-EOL versions it should be fine to not have any conditional logic at all.